src/components/20-molecules/policy-features/policy-features-item/index.js

```diff
@@ -0,0 +1,65 @@
+import { LitElement, html, css, unsafeCSS, svg } from 'lit-element';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Further down below you excluse this rule for the whole file, so you could just use that expression instead of both.